addons/account_edi_ubl_cii/models/account_move.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+from odoo import api, fields, models
+from odoo.addons.account_edi_ubl_cii.models.account_edi_common import COUNTRY_EAS
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    ubl_cii_xml_id = fields.Many2one(
+        comodel_name='ir.attachment',
+        string="UBL/CII Document",
+        copy=False,
+    )
+
+    # --------F-----------------------------------------------------------------
+    # EDI
+    # -------------------------------------------------------------------------
+
+    def _get_ubl_cii_builder(self):
+        self.ensure_one()
+
+        if self.country_code == 'DE':
+            return self.env['account.edi.xml.ubl_de'], {'embed_to_xml': True}
+        if self.country_code in ('AU', 'NZ'):
+            return self.env['account.edi.xml.ubl_a_nz'], {}
```

Comment:
why not `{'embed_to_xml': True}` ? I think it's currently possible to embed the pdf inside the a_nz, no ?